pages/cards/[id].page.jsx

```diff
@@ -18,6 +18,15 @@ function Character({ character, card }) {
   // const character = characters.main.data[characterID];
   const characterLocalizedMain = character.mainLang;
   // console.log(cardsJP);
+
+  const getBreadcrumbs = (path) => {
+    const pathNames = path.split("/");
+    pathNames[
+      pathNames.length - 1
+    ] = `(${cardLocalizedMain.title}) ${characterLocalizedMain.last_name} ${characterLocalizedMain.first_name}`;
```

Comment:
@kamishirorui Do you have a preference for the title in the breadcrumb being the card title and the character name or just the card title?